# HG changeset patch
# User Rik <rik@octave.org>
# Date 1343259039 25200
#      Wed Jul 25 16:30:39 2012 -0700
# Node ID fee211d42c5c985e16aca5f43e279f4763f5336c
# Parent  094bc0a145a180637b5b78a377c2f90f947acb1b
Add demos for plot.m (bug #36939).

* plot.m: Add 3 demos (bug #36939).

diff --git a/scripts/plot/plot.m b/scripts/plot/plot.m
--- a/scripts/plot/plot.m
+++ b/scripts/plot/plot.m
@@ -200,10 +200,25 @@ function retval = plot (varargin)
 
   if (nargout > 0)
     retval = tmp;
   endif
 
 endfunction
 
 
-%% FIXME: Need demo or test for function
+%!demo
+%! x = 1:5;  y = 1:5;
+%! plot (x,y,'g');
+%! title ('plot of green line at 45 degrees');
 
+%!demo
+%! x = 1:5;  y = 1:5;
+%! plot (x,y,'g*');
+%! title ('plot of green stars along a line at 45 degrees');
+
+%!demo
+%! x1 = 1:5;  y1 = 1:5;
+%! x2 = 5:9; y2 = 5:-1:1;
+%! plot (x1,y1,'bo-', x2,y2,'rs-');
+%! axis ('tight');
+%! title ('plot of blue circles ascending and red squares descending with connecting lines drawn'); 
+
